build64/Makefile

```diff
@@ -12,6 +14,151 @@ endif
 CFLAGS = -std=c11 -Wall -Wextra -Wshadow -m64 -march=x86-64 -mno-mmx -mno-sse -mno-sse2 \
          -fpic -fno-builtin -ffreestanding -fomit-frame-pointer -fno-stack-protector
 
+ifeq ($(DEBUG_S), 1)
+  CFLAGS+=-ggdb3
+endif
+
+# In Progress. This option must not be activated as the right flags are not chosen yet.
+ifeq ($(DEBUG_OPT), 1)
+  OPT = -Og
+  OPT_SYSTEM = \
```

Comment:
I agree with @debrouxl , we need to make the code compile with -Og instead of doing this.